front/src/common/Map/Layers/blankStyle.ts

```diff
@@ -13,6 +15,7 @@ const isValidUrl = async (url: string) => {
       const response = await fetch(`${url}.${extension}`);
       return response.ok;
     } catch (error) {
+      setFailure(castErrorToFailure(error));
```

Comment:
This is one of the cases where we don't want to bubble the error to the user, I think.

Reply:
Done